tests/src/Functional/ReplicationSettingsTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Drupal\Tests\workspace\Functional;
+
+use Drupal\replication\ReplicationTask\ReplicationTask;
+use Drupal\Core\Entity\Entity\EntityFormDisplay;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\simpletest\BlockCreationTrait;
+use Drupal\simpletest\BrowserTestBase;
+use Drupal\workspace\ReplicatorManager;
+
+/**
+ * @group workspace
+ */
+class ReplicationSettingsTest extends BrowserTestBase {
```

Comment:
Okay, I think between this test and InternalReplicatorTest, this test will be easier to maintain. As such I want to get this one working and discard the InternalReplicatorTest.

There are two problems with this test:

1. `$page->findButton(t('Save and publish'))->click();` fails because there is no "Save and publish"; I believe I just need to add the workbench moderation module to the test
1.  `drupal_flush_all_caches()` is used and I think it should be removed; the weird part about it is if the $task isn't passed no flush is necessary, e.g. like `$rm->replicate($source_pointer, $target_pointer)`